src/test/custom_nodes/node_perform_different_operations.cpp

```diff
@@ -136,15 +150,22 @@ int getInputsInfo(struct CustomNodeTensorInfo** info, int* infoLength, const str
 }
 
 int getOutputsInfo(struct CustomNodeTensorInfo** info, int* infoLength, const struct CustomNodeParam* params, int paramsLength) {
-    *infoLength = 1;
+    *infoLength = 2;
     *info = (struct CustomNodeTensorInfo*)malloc(*infoLength * sizeof(struct CustomNodeTensorInfo));
-    (*info)->name = "different_ops_results";
-    (*info)->dimsLength = 3;
-    (*info)->dims = (uint64_t*)malloc((*info)->dimsLength * sizeof(uint64_t));
-    (*info)->dims[0] = 1;
-    (*info)->dims[1] = 4;
-    (*info)->dims[2] = 10;
-    (*info)->precision = FP32;
+    info[0]->name = "different_ops_results";
+    info[0]->dimsLength = 3;
+    info[0]->dims = (uint64_t*)malloc((*info)->dimsLength * sizeof(uint64_t));
+    info[0]->dims[0] = 1;
+    info[0]->dims[1] = OPS_END;
+    info[0]->dims[2] = 10;
+    info[0]->precision = FP32;
+    info[1]->name = "factors_results";
+    info[1]->dimsLength = 3;
```

Comment:
Have you tested this call? This will most likely cause segfault since you work on unallocated memory.
To fix the issue, do: `(*info)[0]`/`(*info)[1]`

Reply:
Test metadata calls by running OVMS with library and ping GetMetadata endpoint